scripts/gen_manifests.sh

```diff
@@ -9,7 +9,7 @@ fi
 
 if [ ! -f kn ]; then
   echo "Installing kn-workflow CLI"
-  KN_CLI_URL="https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-linux-amd64.tar.gz"
+  KN_CLI_URL="https://mirror.openshift.com/pub/openshift-v4/clients/serverless/1.11.2/kn-linux-amd64.tar.gz"
```

Comment:
@gabriel-farache pls note this one had to be updated as well.

Reply:
We are duplicating the same piece of code in so many places x))

Reply:
"too many"...only in 3 different places :-P 

but yes, we can invoke a script from the github action, used by the same makefile and reduce one copy.
next, we can evaluate if the pipeline can be modified to use the exact script.

Reply:
> only in 3 different places :-P

Yes but not even in the same repo :)